backends/rapidpro/status.go

```diff
@@ -166,26 +152,22 @@ func (s *StatusUpdate) SetUpdatedURN(old, new urns.URN) error {
 	s.NewURN_ = new
 	return nil
 }
-func (s *StatusUpdate) UpdatedURN() (urns.URN, urns.URN) {
+func (s *StatusUpdate) URNUpdate() (urns.URN, urns.URN) {
 	return s.OldURN_, s.NewURN_
 }
-func (s *StatusUpdate) HasUpdatedURN() bool {
-	if s.OldURN_ != urns.NilURN && s.NewURN_ != urns.NilURN {
-		return true
-	}
-	return false
-}
```

Comment:
generally not good to have something on an interface that every implementation needs to implement in the same way.. it's just forcing code repetition